### PR TITLE
Update NR Customer-Facing Docs Hosting and data storage Sections

### PR DESCRIPTION
**Pull request by demel**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
NR-265888

NR customer facing docs currently make reference to our de-provisioned Chicago hosts, IBM data centers, and state we solely use mysql datastores. The New Relic Database Team has been asked to update the copy to accurately reflect the state of our data storage layers.

* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by demel** [comment]
@homelessbirds

**Comment by akristen** [comment]
Hi @demel Can you sign our CLA? Thanks!

**Comment by demel** [comment]
Hi @akristen, of course. CLA has been signed.

File: src/content/docs/security/security-privacy/data-privacy/security-controls-privacy.mdx [hunk 1]
```diff
@@ -44,11 +44,11 @@ Typically the numbered host is fixed for your account, and it appears in `log/ne
 
 For required changes when you want to create firewall rules to allow the agent to communicate, see [Networks](/docs/using-new-relic/cross-product-functions/install-configure/networks). For more information about security measures for your data in transit to New Relic or at rest in our storage, see [Data encryption](/docs/security/new-relic-security/compliance/data-encryption).
 
-## Hosting and data storage [#Hosting_and_Data_Storage]
+## Cloud Data Storage [#cloud-data-storage]
```
Comment (akristen) [inline src/content/docs/security/security-privacy/data-privacy/security-controls-privacy.mdx:47]: @demel how do you feel about this content update?

**Comment by homelessbirds** [comment]
This is being reviewed by legal right now. Will be done by the end of next week!

**Review by homelessbirds — APPROVED** [review]

**Pull request merged**